src/main/java/org/folio/rest/impl/InvoiceHelper.java

```diff
@@ -0,0 +1,45 @@
+package org.folio.rest.impl;
+
+import static org.folio.invoices.utils.ResourcePathResolver.INVOICES;
+import static org.folio.invoices.utils.ResourcePathResolver.resourcesPath;
+import static org.folio.invoices.utils.HelperUtils.handleGetRequest;
+import static org.folio.invoices.utils.HelperUtils.getEndpointWithQuery;
+
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import me.escoffier.vertx.completablefuture.VertxCompletableFuture;
+
+import org.folio.rest.jaxrs.model.InvoiceCollection;
+
+import io.vertx.core.Context;
+
+public class InvoiceHelper extends AbstractHelper {
```

Comment:
The formatting is a little off for this file

Reply:
I opened this file in View mode and can't find any formatting issue. Can you point me to those lines?

Reply:
![Screen Shot 2019-04-26 at 10 42 38 AM](https://user-images.githubusercontent.com/41596468/56815872-57019e00-6810-11e9-9f64-aeb6c2ec92c8.png)

This is how it looks on the IDE, so the thenAccept and exceptionnaly blocks and the class in general with spacing

Reply:
I don't see that in my IDE. Every IDE can have different formatting options set. This is how it looks in git view which should be the same for everyone:
![image](https://user-images.githubusercontent.com/17807360/56816172-fb83e000-6810-11e9-947b-1d6b28b51699.png)

Reply:
That is true, but the actual reason this is happening because few lines before L31 are having tabs and the other lines have spaces, and the formatting must have only spaces according to folio conventions. It shows it is not formatted in the "Files Changed" tab

Reply:
I have set 2 spaces = 1 tab in my IDE config setting. I would like to press tab twice instead of pressing space 4 times :)
I have pushed one change, not sure if that fixes what you are seeing. I don't see the issue in my IDE.

Reply:
yup, the issue is not IDE specific, its just because of tabs and spaces. Looks good now